ecllibrary/std/Str.ecl

```diff
@@ -305,19 +305,21 @@ EXPORT STRING CombineWords(SET OF STRING words, STRING separator) := lib_stringl
  * 
  * @param _left         The first string to be compared.
  * @param _right        The second string to be compared.
- * @return              The minimum edit distance between the two strings.
+ * @param radius        The maximum edit distance that is acceptable, or 0 for no limit.  Defaults to 0.
+ * @return              The minimum edit distance between the two strings.  Edit distances above radius will
+                        return a value larger than radius.
```

Comment:
perhaps should say "an arbitrary value larger than radius"

Reply:
changed, and in uni.ecl